<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>645</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_find">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Поиск</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="TB_tools">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Сообщения</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableWidget" name="TW_messages">
          <property name="minimumSize">
           <size>
            <width>1060</width>
            <height>0</height>
           </size>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>200</number>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>200</number>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Номер сообщения</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Дата регистрации</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Тип происшествия</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Решение</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Регистрационный номер дела</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="PB_m_entry">
            <property name="minimumSize">
             <size>
              <width>900</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>900</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Добавить запись</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PB_sort">
            <property name="text">
             <string>Отсортировать по дате</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Лица</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="TW_persons">
          <property name="minimumSize">
           <size>
            <width>1060</width>
            <height>0</height>
           </size>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="showGrid">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>166</number>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>166</number>
          </attribute>
          <column>
           <property name="text">
            <string>Регистрационный номер</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Имя</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Фамилия</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Отчество</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Адрес</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Кол-во судимостей</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <kerning>true</kerning>
            </font>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PB_p_entry">
          <property name="text">
           <string>Добавить запись</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Отношение</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableWidget" name="TW_relations">
          <property name="minimumSize">
           <size>
            <width>1060</width>
            <height>0</height>
           </size>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>333</number>
          </attribute>
          <column>
           <property name="text">
            <string>Регистрационный номер сообщения</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Регистрационный номер лица</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Отношение к происшествию</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PB_r_entry">
          <property name="text">
           <string>Добавить запись</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
